ntc_templates/templates/alcatel_aos_show_interfaces_port.textfsm

```diff
@@ -0,0 +1,25 @@
+Value PORT ((\d+\/?)+)
```

Comment:
Yep, I'm backtracking to a simpler regex. This also matches the PORT regex used by other AOS templates.

```suggestion
Value PORT (\S+)
```